ads/google/a4a/test/test-traffic-experiments.js

```diff
@@ -76,26 +71,26 @@ describe('all-traffic-experiments-tests', () => {
 
     it('handles empty experiments list', () => {
       // Opt out of experiment.
-      sandbox.win.AMP_CONFIG['testExperimentId'] = 0.0;
+      toggleExperiment(sandbox.win, 'testExperimentId', false, true);
```

Comment:
No, I agree that you shouldn't have to touch AMP_CONFIG.  But this experiment _does_ do more than just care whether the experiment is on.  `randomlySelectUnsetPageExperiments` does what its name says: it rolls the dice for each requested experiment.  The test is verifying that it sets the experiment value as expected _given_ that the experiment probability is a certain value.  In this change, you're setting the experiment value before calling `randomlySelectUnsetPageExperiments`.  That method will look and say, "Oh, that experiment already has a value.  I won't touch it."  So you're no longer testing its behavior, which is really the point of this test.